flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/expressions/converter/OverConvertRule.java

```diff
@@ -182,57 +191,59 @@ private RexNode createCollation(
     }
 
     private RexWindowBound createBound(ConvertContext context, Expression bound, SqlKind sqlKind) {
-        if (bound instanceof CallExpression) {
-            CallExpression callExpr = (CallExpression) bound;
-            FunctionDefinition func = callExpr.getFunctionDefinition();
-            if (BuiltInFunctionDefinitions.UNBOUNDED_ROW.equals(func)
-                    || BuiltInFunctionDefinitions.UNBOUNDED_RANGE.equals(func)) {
+        if (bound instanceof ValueLiteralExpression) {
+            final ValueLiteralExpression literal = (ValueLiteralExpression) bound;
+            return literal.getValueAs(OverWindowRange.class)
+                    .map(r -> createSymbolBound(r, sqlKind))
+                    .orElseGet(
+                            () ->
+                                    createLiteralBound(
+                                            context, (ValueLiteralExpression) bound, sqlKind));
+        } else {
+            throw new TableException("Unexpected expression: " + bound);
```

Comment:
Have we tested this case?

(I'm asking to get an idea how this is called.)

Reply:
No, we have not. It should not be possible to reach this line. This is a) precaution b) solves checkstyle warning